pySDC/tests/test_benchmarks/test_collocation.py

```diff
@@ -25,6 +25,7 @@ def wrapper():
     benchmark(wrapper)
 
 
+@pytest.mark.base
```

Comment:
@tlunet, do you agree with this markers? I don't think we need to run these tests in all environments. They are throwing some numpy deprecation warnings at the moment, which we will need to deal with someday.

Reply:
They are necessary. You just need to replace in the assertion the == between arrays by some np.allclose (as the warning is suggesting ?)

Reply:
That's a polynomial deprecation warrning ?

Reply:
I didn't pay much attention to the warning, tbh. For now, I just want to run these tests only in the base environment and not in every environment.

Reply:
Oh, then that's fine indeed ... sorry I didn't realized that was an addition :sweat_smile: